site/components/map/ServicesMap.tsx

```diff
@@ -484,19 +488,32 @@ const Map = ({
               )} (${service.operatorShortName})`
             : "Line: N/A";
     };
+    useEffect(() => {
```

Comment:
Since we're already doing this check in the API, rather than import the library here as well and do the same check it might be nicer to just look at the api response for the "Area of polygon must be below" error and handle it that way

Reply:
done